src/material-experimental/tree/data-source/nested-data-source.ts

```diff
@@ -28,7 +28,7 @@ export class MatTreeNestedDataSource<T> implements DataSource<T> {
   set data(value: T[]) { this._data.next(value); }
 
   connect(collectionViewer: CollectionViewer): Observable<T[]> {
-    return merge([collectionViewer.viewChange, this._data])
+    return merge(...[collectionViewer.viewChange, this._data])
```

Comment:
this is the same as `merge(collectionViewer.viewChange, this._data)` right?

Reply:
Yes